improver_tests/acceptance/SHA256SUMS

```diff
@@ -770,6 +770,61 @@ db89fbab53988556f708cbffaa7639b153ffa81eff4f5586398e7efa90af9db2  ./phase-probab
+8ca0d1d7891b080e7af879e7ece0b4137536f9cd138007ab98eb2848b08a1d2f  ./precipitation_duration/spot_data/20250128T0900Z-precipitation_accumulation.nc
+752414ed48ce241caad82b2bab7d3ab13066d697c8c018b3507863f43745e390  ./precipitation_duration/spot_data/20250128T0900Z-precipitation_rate_max.nc
+fab03bfed8e54384737f9437d3e31d2d1e1647db23dd81f4e879f1186a3e31a2  ./precipitation_duration/spot_data/kgo.nc
+3e7298a5c37d73e4faa2f3f640a27e8fe28fa705fc87806b82fe1163873b0a70  ./precipitation_duration/standard_names/20250127T1200Z-precipitation_accumulation.nc
+487a50f8babf2fb37e0aa8263729fae38e640d6002941903a52e21b293f509bc  ./precipitation_duration/standard_names/20250127T1200Z-precipitation_rate_max.nc
+0b18ac8c7d1a4c6f0ee6c8952ab3f7dccba0aaa81631c59143200aa137ce5dee  ./precipitation_duration/standard_names/20250127T1500Z-precipitation_accumulation.nc
+6a881933afb383eacbfa262c0919054b8774ac6975e93530387e1ff1f63ed80a  ./precipitation_duration/standard_names/20250127T1500Z-precipitation_rate_max.nc
+338bf0caa4eeae13762489121fb833eefee04887931b0f74320f771e11fd8dc6  ./precipitation_duration/standard_names/20250127T1800Z-precipitation_accumulation.nc
+4c6d95c3909aa6fbea46b568d336d40258ce8b910d240bc897e41eaf7f2e53ca  ./precipitation_duration/standard_names/20250127T1800Z-precipitation_rate_max.nc
+d1c18f5e1da3728f6963489370807dcb24fa1cd975f8e39987f29f6945765562  ./precipitation_duration/standard_names/20250127T2100Z-precipitation_accumulation.nc
+911b28d3a72616c3cec4b11c02a490f0f2d4aeb908d74cbcb68d666513664e6c  ./precipitation_duration/standard_names/20250127T2100Z-precipitation_rate_max.nc
+f090d365cb7264654a8df937f95fbd2ca8026d9b3c6cc8f092dfdaf54441549d  ./precipitation_duration/standard_names/20250128T0000Z-precipitation_accumulation.nc
+e06f5b6876144b5a0f211bdecc9047340a9f53b0a89e0b2b83207dee5dad27b5  ./precipitation_duration/standard_names/20250128T0000Z-precipitation_rate_max.nc
+7aeafa66500c6c9277176d883ad6837d40065a25c3dd33ce7bbc9cf32bdacc8b  ./precipitation_duration/standard_names/20250128T0300Z-precipitation_accumulation.nc
+6b6bcab08248c5e4d1b0d2e46d9fcdc9ad7ab2ce356a503ac8834c6be58c6e7d  ./precipitation_duration/standard_names/20250128T0300Z-precipitation_rate_max.nc
+84a263cc4f4a2233ebcdea57cfa39afb3c13bede245817256765186ff00cc09e  ./precipitation_duration/standard_names/20250128T0600Z-precipitation_accumulation.nc
+97dcccb976305aa1510eee6713b1346ce425b122058012c9034f47646df52ceb  ./precipitation_duration/standard_names/20250128T0600Z-precipitation_rate_max.nc
+a5666723c4b27f7f27b05e1c1dfca396bab7661e06b6ecc2458a36879c5949d1  ./precipitation_duration/standard_names/20250128T0900Z-precipitation_accumulation.nc
+a89ba9668fd878ed5c5cc017e46a25ab1f9d205b1a6913457a8f3af770cc49e1  ./precipitation_duration/standard_names/20250128T0900Z-precipitation_rate_max.nc
+7c95bd10ff3330bb68cc9ba161a25081593b0e2b6f0dec174d181f19d04f7b42  ./precipitation_duration/standard_names/hmm.nc
```

Comment:
I think that this file should probably be removed, as it doesn't seem to be the input or output for any acceptance test.

Reply:
And has my standard silly name of `hmm`, so I really should have noticed.